### ISSUE TITLE
Studio0x: A decentralized film and TV studio and distributor

### ISSUE THREAD

**Issue by RequiemForTheFuture** [district0x/district-proposals#208]
## **Name**: 
Studio0x: A decentralized film and TV studio and distributor.

## **Purpose**: 
Studio0x aims to provide independent filmmakers and video content creators a platform and marketplace to fund and sell their content. Essentially, Studio0x is decentralized film and television content funding, creation and distribution. 

The fundamental reason for this district is to offer creators a way to cut out the middleman and be paid directly by consumers or advertisers. The long term goal is to create a community run, decentralized and mass-adopted competitor to Netflix, Disney Studios, YouTube and other television networks (***see first comment for a few additional details.) This project, if successfully implemented, could have sky-high potential.

Media projects exist on a variety of budget levels and the district eventually should be able to support any level of production.

## **Description**: 
Studio0x is a method to fund and distribute films, tv and other forms of entertainment. Studio0x platform is accessible to everyone- you do not have to be a staking DNT to access its content. The primary point of contact with Studio0x is a web site (or app) consisting of two main areas: the viewer hub, the creator hub and the curator hub.
There are five member levels and corresponding hubs within Studio0x: viewers, creators, investors, advertisers and curators.

**Viewers/Viewer Hub:**
Viewers can purchase on-demand on content through the District Box Office with ETH or DNT or view ad-supported content free-of-charge (the content creator can choose whether content is ad supported or pay per view or both.) Viewers can create watch-lists or share content with other people. The viewing hub is searchable and sortable based on various metadata (IE genre) and  has subsections for various formats: the District box office (DBO), DistrictTV, etc.

**Creators/Creators Hub:**
Content creators are the backbone of this district. They use the district to fund and monetize their work: movies, TV shows, webisodes, podcasts, etc. To register as a creator you must pay a DNT fee. If feasible, ID verification (through, say, Civic or something similar) should be implemented for creators. The purpose of the fee and verifiable ID is to prevent abuses such as piracy or child pornography or outright scams.
If creator wants to fund a film or other project through the district, creators submit a proposal for the project with all relevant details: budget, media format, cast, crew, salaries and other estimated production costs, profit sharing for investors. As in evidenced in current film and tv funding, a content creator’s reputation is important to investors. This is secondary function of ID verification for creators.  Essentially, content creators can create a business proposal of their vision to attract investors. Funds could be raised in DNT or ETH smart contracts. The creator can update investors through their portal on the status of the project as it progresses. Once the project is complete, the creator submits the content to the platform, similar to YouTube. Once the content is live, income is earned through advertising and pay-per-view based contracts.  Creators can predetermine profit sharing with any investors, cast, the production staff or even charities, utilizing smart contracts and executed with ETH. Additionally, a small percentage of profits goes back to staking district members, curators, as well as a portion going to unfunded projects. Through this, the studio0x platform will be continuously funding content simply by being used.

If a piece of content is self-funded or already exists, it could be added directly to the District box office or DistrictTV channel. 

There is an additional DNT fee when submitting content to curation. This further disincentives submitting inappropriate content.

Fees paid by creators are redistributed to DNT members, unfunded DNT projects and curators.

Creators can pay a DNT fee to promote their content on the splash page.

**Investors/Investor Hub:**
Investors fund projects that interest them or they believe to be good investment opportunities based on profit sharing or community interest. They utilize the creators platform to view potential projects and research the individuals involved. Investors can enjoy profit sharing based on smart contracts. The profit sharing would be predetermined by the creator, and readily available in the creators proposal. Investors view and contribute to projects on the investor hub. Investors submit their investment via ETH or DNT. 

District members staking DNT can upvote potential projects (without spending any DNT or ETH), which directs a small portion of the district fees to that project. It also signals to investors which projects viewers are interested in.
 
**Advertisers Hub:**
Advertisers are optional way to monetize content. A separate marketplace would be created to link up creators and advertisers. The majority of advertising revenue would go directly to the creators and their investors (as determined by the creators), with small percentage of advertising revenue would go to curators, unfunded projects staking DNT members..  Advertisers pay with DNT or ETH utilizing smart contracts. Advertisers could have access to various types of metadata to link them with the proper content creators. Additionally, they can request various different types of ad buys from creators: 15 second, 30 second, 
pre-roll, mid-roll, post-roll, 1 month buys, specific times, etc.

**Curators:**
The most difficult problem for me to solve has been that of piracy and other abuses. My current solution is through curators, though perhaps there is a more elegant solution lurking out there. This is slightly modeled on Augur, but I have not done incredibly deep dive on this so it may not even be possible utilizing DNT. 

The point of the curator is to prevent abuses (piracy, etc) and maintain the accuracy of the metadata and database. All content going live must be verified by multiple curators that it meets community standards. Curators get paid for their work in DNT through the revenue collected by Studio0x. Members staking DNT can submit to become curators. Curators must put DNT into an escrow smart contract. If a curator deviates from final consensus, they are fined DNT. If curators do not report, they are fined DNT. 

Viewers can request curators re-examine content already live on the platform and put it back into the curation queue. 

One potential exploit could be to have a whale manipulate curation, with the intention of stifling competing projects. Solution TBD.

**Additional benefits of staking:**
In addition to collecting a percentage of profits, staking DNT members can also vote on projects or creators they're interested in, signaling interest to potential investors. Staking token holders can also upvote content they enjoy, or even curate channels/viewing lists. 

**A note on potential illegal or otherwise illicit activities:**
As mentioned above, Studio0x is not about piracy, scams, child pornography or any other illicit activity. The function of the fees, ID verification and curators are that they are all methods of preventing such activities. The point is to reward the actual creators, and not let John Doe submit and profit off of season 1 of Game of Thrones.

## **Ethereum Address**:
0x0e4Ea589a6332355cc8A79c16688437AF443C5dF

**Comment by RequiemForTheFuture** [comment]
***CMA/Notes:
I’m a crytpo enthusiast, not a crypto professional or even a programmer. There are many aspects to the tech side of crypto that I am ignorant of. I do, however, work with video content (post-production film, tv, advertising) and am aware of the massive amount of bandwidth something like this would take, and additionally aware that there are limitations on the ethereum blockchain. However, I’m not enough of techie to know whether overcoming these obstacles is within Ethereum’s roadmap.

**Comment by Kyle-Kidd** [comment]
This is an excellent proposal.  Let's do it!

**Comment by diop** [comment]
This is actually not a bad idea. What's also missing is a more decentralized IMDB for film production credits.

**Comment by RequiemForTheFuture** [comment]
Yes, some sort of IMDb-like integration seems like a natural fit.

**Comment by Bradymck** [comment]
Hey @RequiemForTheFuture 

I'm not sure if you're aware but we recently launched a new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.
